radis/test/spectrum/test_compare.py

```diff
@@ -66,6 +67,20 @@ def test_compare_methods(verbose=True, plot=True, close_plots=True, *args, **kwa
 
 # %%
 
+def test_plotdiff_nan(verbose=True, plot=True, close_plots=True, *args, **kwargs)
+    s = calc_spectrum(1900, 2300,         # cm-1
+                   molecule='CO',
+                   isotope='1,2,3',
+                   pressure=1.01325,   # bar
+                   Tgas=700,           # K
+                   mole_fraction=0.1,
+                   path_length=1,      # cm
+                   )
+    
+    s=Radiance_noslit(s)  
+    s._q['radiance_noslit'][0] = np.nan
+
+    plot_diff(s, s*1.2, normalize=True)
```

Comment:
in the test can you also try some other normalization options? Like, "max", "area", etc.

Reply:
Sure

Reply:
Hi, 
You even added a test! Very good :)
I just suggest you remove the lines you commented.
Minou

Reply:
> Hi,
> You even added a test! Very good :)
> I just suggest you remove the lines you commented.
> Minou

Hey there, I did what you suggested and pushed changes. I also used black to format the code, hope the code quality checks pass this time.

Reply:
> in the test can you also try some other normalization options? Like, "max", "area", etc.

I'm a bit confused about that, I've tried doing setting `normalization_how` manually, and it works, but how can I write a method to test that? Since `normalization_how` is an argument of the get_residual method, and that method is called from `plot_diff` method, and there's no `normalization_how` argument in that.

Reply:
You're right! So far I'd include a `get_residual(s)` test line in the test file! 

@minouHub do we want to have more normalization options in plot_diff ? ("normalize=" could be given the "area" keywords, or a tuple to normalize on a certain range, like in [plot](https://radis.readthedocs.io/en/latest/source/radis.spectrum.spectrum.html#radis.spectrum.spectrum.Spectrum.plot) ). In any case that would be a separate Issue!

Reply:
Well, I've never used that in post-processing, so I'm not sure it is necessary.